Conceptpower+Spring/src/main/java/edu/asu/conceptpower/core/ConceptEntry.java

```diff
@@ -29,43 +29,43 @@
     private static final long serialVersionUID = 4569090620671054560L;
 
     @Id
-    @LuceneField(lucenefieldName = LuceneFieldNames.ID, isIndexable = false)
+    @LuceneField(lucenefieldName = LuceneFieldNames.ID, isTokenized = false)
     private String id;
 
     @SearchField(fieldName = SearchFieldNames.WORDNETID)
-    @LuceneField(lucenefieldName = LuceneFieldNames.WORDNETID, isIndexable = true)
+    @LuceneField(lucenefieldName = LuceneFieldNames.WORDNETID, isTokenized = false)
     private String wordnetId;
 
     @SearchField(fieldName = SearchFieldNames.WORD)
-    @LuceneField(lucenefieldName = LuceneFieldNames.WORD, isIndexable = true)
+    @LuceneField(lucenefieldName = LuceneFieldNames.WORD, isTokenized = true)
     private String word;
 
     @SearchField(fieldName = SearchFieldNames.DESCRIPTION)
-    @LuceneField(lucenefieldName = LuceneFieldNames.DESCRIPTION, isIndexable = true)
+    @LuceneField(lucenefieldName = LuceneFieldNames.DESCRIPTION, isTokenized = true)
     private String description;
 
     @SearchField(fieldName = SearchFieldNames.POS)
-    @LuceneField(lucenefieldName = LuceneFieldNames.POS, isIndexable = true)
+    @LuceneField(lucenefieldName = LuceneFieldNames.POS, isTokenized = true)
     private String pos;
 
     @SearchField(fieldName = SearchFieldNames.CONCEPT_LIST)
-    @LuceneField(lucenefieldName = LuceneFieldNames.CONCEPT_LIST, isIndexable = false)
+    @LuceneField(lucenefieldName = LuceneFieldNames.CONCEPT_LIST, isTokenized = false)
     private String conceptList;
 
     @SearchField(fieldName = SearchFieldNames.TYPE_ID)
-    @LuceneField(lucenefieldName = LuceneFieldNames.TYPE_ID, isIndexable = false)
+    @LuceneField(lucenefieldName = LuceneFieldNames.TYPE_ID, isTokenized = false)
     private String typeId;
 
     @SearchField(fieldName = SearchFieldNames.EQUALS_TO)
-    @LuceneField(lucenefieldName = LuceneFieldNames.EQUALS_TO, isIndexable = true)
+    @LuceneField(lucenefieldName = LuceneFieldNames.EQUALS_TO, isTokenized = false)
     private String equalTo;
 
     @SearchField(fieldName = SearchFieldNames.SIMILAR_TO)
-    @LuceneField(lucenefieldName = LuceneFieldNames.SIMILAR_TO, isIndexable = true)
+    @LuceneField(lucenefieldName = LuceneFieldNames.SIMILAR_TO, isTokenized = false)
     private String similarTo;
 
     @SearchField(fieldName=SearchFieldNames.SYNONYM_ID)
-    @LuceneField(lucenefieldName = LuceneFieldNames.SYNONYMID, isIndexable = true)
+    @LuceneField(lucenefieldName = LuceneFieldNames.SYNONYMID, isTokenized = false)
```

Comment:
This is not a very clean approach. Basically, what you're saying is either a field is tokenized (then create tokens out of a field's contents) or it's not then split the fields content by ','. But what if we have a field that shouldn't be tokenized but does contains a comma and it's not a list. Eg: we add a field to enter normalized names ("Einstein, Albert"). In other words, you should not merge two separate things: tokenization and multi-valued fields.